api/services/AuthorizationPolicy/index.js

```diff
@@ -118,10 +118,16 @@ policy.include('calendarEventInvite', (p) => {
     ({ viewer, entity: { event, status }, action }) =>
       isOrganiser(event, viewer) ||
       isAdmin(viewer) ||
-      (isUser(viewer) &&
-        (status === 'Requested' || status === 'Accepted') &&
+      (isMember(viewer) &&
         action === ':create' &&
-        event.is_requestable),
+        status === 'Requested' &&
+        action === ':create' &&
+        event?.is_requestable) ||
+      (isMember(viewer) &&
+        action === ':create' &&
+        status === 'Accepted' &&
+        action === ':create' &&
+        event?.auto_accept_requests),
```

Comment:
Improve readability: 
```javascript
isUser(viewer) 
&& action === ':create' 
&& event?.is_requestable 
&& (status === 'Requested' || status === 'Accepted' && event?.auto_accept_requests)
```

Reply:
this has been changed .if you entered via  `View changes` button .it doesnt show up because i force pushed.you can see the difference in files.

Reply:
Explain me how this is similar to what I suggested.

![Screenshot 2020-05-28 at 4 19 19 PM](https://user-images.githubusercontent.com/22571395/83132432-04736f00-a0ff-11ea-9290-e30a783aedd4.png)